api/tests/unit/app_analytics/split_testing/test_unit_split_testing_tasks.py

```diff
@@ -0,0 +1,139 @@
+from app_analytics.models import FeatureEvaluationRaw
+from app_analytics.split_testing.models import ConversionEvent, SplitTest
+from app_analytics.split_testing.tasks import _update_split_tests
+from pytest_django.fixtures import SettingsWrapper
+
+from environments.identities.models import Identity
+from environments.models import Environment
+from features.models import Feature
+from features.multivariate.models import MultivariateFeatureOption
+from features.value_types import STRING
+from projects.models import Project
+
+
+def test_update_split_tests(
+    settings: SettingsWrapper,
+    project: Project,
+    environment: Environment,
+) -> None:
+    # Given
+    settings.USE_POSTGRES_FOR_ANALYTICS = True
+
+    feature1 = Feature.objects.create(
+        name="1",
+        project=project,
+        initial_value="200",
+        is_server_key_only=True,
+        default_enabled=True,
+    )
+    feature2 = Feature.objects.create(
+        name="2",
+        project=project,
+        initial_value="banana",
+        default_enabled=False,
+    )
+
+    MultivariateFeatureOption.objects.create(
+        feature=feature1,
+        default_percentage_allocation=30,
+        type=STRING,
+        string_value="mv_feature_option1",
+    )
+    MultivariateFeatureOption.objects.create(
+        feature=feature1,
+        default_percentage_allocation=70,
+        type=STRING,
+        string_value="mv_feature_option2",
+    )
+    MultivariateFeatureOption.objects.create(
+        feature=feature2,
+        default_percentage_allocation=50,
+        type=STRING,
+        string_value="mv_feature_option3",
+    )
+    MultivariateFeatureOption.objects.create(
+        feature=feature2,
+        default_percentage_allocation=50,
+        type=STRING,
+        string_value="mv_feature_option4",
+    )
```

Comment:
It's odd to me that we're not using the control value at all here. Both features have multivariate feature options defined that add up to 100%. This doesn't really seem like a realistic use case that we're testing.

Reply:
From the call with @matthewelwell there is a fallback that I missed in the last line of `get_multivariate_feature_state_value` where the default value is selected if there isn't a matching multivariate feature state value.

The fix for this will necessitate adding in a null value to the multivariate feature option and falling back to the feature value for those records, which should be one per collection of feature, environment, and mv feature option.